hack/pin-custom-bundle-dockerfile.sh

```diff
@@ -39,7 +39,8 @@ for MOD_PATH in $(go list -m -json all | jq -r '. | select(.Path | contains("ope
     fi
 
     if [[ ${LOCAL_REGISTRY} -eq 1 && "$BASE" == "$IMAGEBASE" ]]; then
-        SHA=$(curl -s ${REPO_CURL_URL}/$BASE-operator-bundle/tags/list | jq -r .tags[] | sort -u | grep $REF)
+        SHA=$(curl -s ${REPO_CURL_URL}/$BASE-operator-bundle/tags/list | tee tags | jq -r .tags[] | sort -u | grep $REF)
+        cat tags
```

Comment:
there is something wrong with the local registry support
this fails to find the nova-opertor and i expect it would fail for any operators built and pushed to the local regisry 

im wondering is there an auth issue since based on https://docs.docker.com/registry/spec/api/#get-tags this looks right so im teeing the responce to tag and cating it to see what the responce has

Reply:
oh its not taking this branch its using the other barnch 
https://review.rdoproject.org/zuul/build/762415ad6f434a07a5bfb912beadc42a/log/ci-framework/logs/ci_make_011_openstack-operator_-_call_bundle.log#314-315